### PR TITLE
Fix requirements with Symfony 6.x

### PR DESCRIPTION
**Pull request by bytehead**
```
  Problem 1
    - symfony-cmf/routing[2.3.0, ..., 2.3.2] require php ^7.2 -> your php version (8.1.8) does not satisfy that requirement.
    - symfony-cmf/routing[2.3.3, ..., 2.3.4] require symfony/routing ^4.4 || ^5.0 -> found symfony/routing[v4.4.0, ..., v4.4.41, v5.0.0, ..., v5.4.8] but it conflicts with your root composer.json require (^6.0).
    - contao/core-bundle 5.0.0-RC1 requires symfony-cmf/routing ^2.3 -> satisfiable by symfony-cmf/routing[2.3.0, ..., 2.3.4].
    - Root composer.json requires contao/core-bundle ^5.0.0-RC1 -> satisfiable by contao/core-bundle[5.0.0-RC1].
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -98,8 +98,8 @@
         "simplepie/simplepie": "^1.3",
         "spatie/schema-org": "^3.4",
         "spomky-labs/otphp": "^10.0",
-        "symfony-cmf/routing": "^2.3",
-        "symfony-cmf/routing-bundle": "^2.4",
+        "symfony-cmf/routing": "^2.3 || ^3.0",
+        "symfony-cmf/routing-bundle": "^2.4 || ^3.0",
```
Comment (leofeyer) [inline composer.json:102]: Does it make sense to keep the old versions if they are not installable anyway?

Comment (bytehead) [reply composer.json:102]: They are with Symfony 5.x.

Comment (leofeyer) [reply composer.json:102]: I see.

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Pull request merged**